### DISCUSSION TITLE
"This command must be executed inside a Quasar v1+ project folder."

### DISCUSSION THREAD

**Discussion by 406rend** [quasarframework/quasar#15205] [General - Components / Directives / etc] [answered]
I have built quasar apps before. I am creating a new one, and I wanted to use a couple pages from another project, so I copied them over. I believe I also initialized firebase. I tried the command "quasar dev" again, and I got the error "App • ⚠️  Error. This command must be executed inside a Quasar v1+ project folder." quasar info gives the same error. I am in the correct project. When I run the command "init quasar" I get the error that this command may NOT be ran in a quasar project. 

Here is my project.json:

{
  [name redacted]
  "private": true,
  "scripts": {
    "lint": "eslint --ext .js,.vue ./",
    "test": "echo \"No test specified\" && exit 0"
  },
  "dependencies": {
    "@quasar/extras": "^1.0.0",
    "cli": "^1.0.1",
    "core-js": "^3.6.5",
    "pinia": "^2.0.11",
    "quasar": "^2.11.3",
    "vue": "^3.0.0",
    "vue-router": "^4.0.0"
  },
  "devDependencies": {
    "@babel/eslint-parser": "^7.13.14",
    "@quasar/app-webpack": "^3.0.0",
    "eslint": "^8.10.0",
    "eslint-config-standard": "^17.0.0",
    "eslint-plugin-import": "^2.19.1",
    "eslint-plugin-n": "^15.0.0",
    "eslint-plugin-promise": "^6.0.0",
    "eslint-plugin-vue": "^9.0.0",
    "eslint-webpack-plugin": "^3.1.1"
  },
  "browserslist": [
    "last 10 Chrome versions",
    "last 10 Firefox versions",
    "last 4 Edge versions",
    "last 7 Safari versions",
    "last 8 Android versions",
    "last 8 ChromeAndroid versions",
    "last 8 FirefoxAndroid versions",
    "last 10 iOS versions",
    "last 5 Opera versions"
  ],
  "engines": {
    "node": ">= 12.22.1",
    "npm": ">= 6.13.4",
    "yarn": ">= 1.21.1"
  }
}

**Comment by metalsadman** [comment]
reinstall your dependencies.

**Reply by 406rend** [reply]
Thanks, @metalsadman.

I just realized when I "Picked the quasar version" during install, I selected "Quasar v2" -- is this the issue?

 I've gone through one by one, and I'm still having the issue. Is there an approved way I should reinstall them?


updated package.json below
{
[  omitted]
  "scripts": {
    "dev": "quasar dev",
    "build": "quasar build",
    "lint": "eslint --ext .js,.vue ./",
    "test": "echo \"No test specified\" && exit 0"
  },
  "dependencies": {
    "@quasar/cli": "^1.3.2",
    "@quasar/extras": "^1.15.9",
    "core-js": "^3.27.1",
    "dependencies": "^0.0.1",
    "firebase": "^9.15.0",
    "pinia": "^2.0.28",
    "postcss-loader": "^7.0.2",
    "quasar": "^2.11.3",
    "vue": "^3.2.45",
    "vue-advanced-cropper": "^2.8.6",
    "vue-router": "^4.1.6",
    "vuex": "^4.1.0"
  },
  "devDependencies": {
    "@babel/eslint-parser": "^7.19.1",
    "@quasar/app-webpack": "^3.6.2",
    "eslint": "^8.10.0",
    "eslint-config-standard": "^17.0.0",
    "eslint-plugin-import": "^2.19.1",
    "eslint-plugin-n": "^15.0.0",
    "eslint-plugin-promise": "^6.0.0",
    "eslint-plugin-vue": "^9.0.0",
    "eslint-webpack-plugin": "^3.1.1"
  },
  "browserslist": [
    "last 10 Chrome versions",
    "last 10 Firefox versions",
    "last 4 Edge versions",
    "last 7 Safari versions",
    "last 8 Android versions",
    "last 8 ChromeAndroid versions",
    "last 8 FirefoxAndroid versions",
    "last 10 iOS versions",
    "last 5 Opera versions"
  ],
  "engines": {
    "node": ">= 12.22.1",
    "npm": ">= 6.13.4",
    "yarn": ">= 1.21.1"
  }
}

**Reply by metalsadman** [reply]
run npm install / yarn

**Reply by 406rend** [reply]
This hasn't worked yet. Continuing to try other options.

**Comment by 406rend** [comment, marked as answer]
Answer: Changed quasar.config.js to quasar.conf.js

Not sure why that is the problem or why it's pushed like that. https://github.com/quasarframework/quasar/issues/13771
